doc/operation-and-maintenance/Cluster-configuration-and-node-management.md

```diff
@@ -2,15 +2,15 @@
 
 ### File descriptors
 
-To handle large traffic, some of the system variables need to be tuned. 
-Number one on that list is the maximum number of file descriptors which by default is set to 1024. 
+To handle large traffic, some of the system variables need to be tuned.
+Number one on that list is the maximum number of file descriptors which by default is set to 1024.
```

Comment:
Does this consider both systemd and non-systemd systems?

Reply:
Indeed I would rather say "which e.g. in many Linux systems is set to 2014."